tasks/managed/update-trusted-tasks/update-trusted-tasks.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-trusted-tasks
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update trusted-tasks list bundles
+  params:
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the Snapshot spec in the data workspace
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot json reside
+  steps:
+    - name: update-trusted-tasks
+      image: quay.io/konflux-ci/appstudio-utils@sha256:0901e46d9a41d0c564b0efab092f37cd187bd8b4ff2455ec84539ceb8b08f3ea
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+        TAG="latest"
+
+        SNAPSHOT_SPEC_FILE="$(workspaces.data.path)/$(params.snapshotPath)"
+        if [ ! -f "${SNAPSHOT_SPEC_FILE}" ] ; then
+            echo "No valid snapshot file was found."
+            exit 1
+        fi
+
+        application=$(jq -r '.application' "${SNAPSHOT_SPEC_FILE}")
+        NUM_COMPONENTS=$(jq '.components | length' "${SNAPSHOT_SPEC_FILE}")
+        printf 'Beginning "%s" for "%s"\n\n' "$(context.task.name)" "$application"
+        for ((i = 0; i < NUM_COMPONENTS; i++))
+        do
+            component=$(jq -c --argjson i "$i" '.components[$i]' "${SNAPSHOT_SPEC_FILE}")
+            repository=$(jq -r '.repository' <<< "$component")
+            imageTag=$(jq '.tags' <<< "$component" | jq -r '.[0]')
+            ACCEPTABLE_BUNDLES=$(echo "${repository}" | awk -F'/' '{$NF="data-acceptable-bundles"; print $0}' OFS='/')
```

Comment:
I had a hard time figuring out what this was. Maybe add a comment saying that this replaces the repo with "data-acceptable-bundles", e.g. quay.io/myorg/myrepo will be quay.io/myorg/data-acceptable-bundles.

Reply:
I will add a comment to explain it

Reply:
@mmalina I updated the description and added comment, I hope its clearer now

Reply:
Thanks.